library/src/main/java/com/macroyau/thingspeakandroid/ThingSpeakChannel.java

```diff
@@ -89,7 +89,7 @@
 
     }
 
-    private static final String THINGSPEAK_API = "https://api.thingspeak.com";
+    private static String THINGSPEAK_API = "https://api.thingspeak.com";
```

Comment:
You do not really need to change the modifier of this constant. This is only accessed once in the constructor, and does not affect supporting custom endpoints.